src/CodePen.js

```diff
@@ -0,0 +1,15 @@
+import { URL } from 'url';
+
+export const shouldTransform = url => {
+  const urlObj = new URL(url);
+  if (urlObj.host === 'codepen.io') {
+    return !!urlObj.pathname.match(/\/pen\/.*\w/);
```

Comment:
I don't think a regex is the best way of handling this.

Have a look at the Twitter transformer for how I think we should handle this one
https://github.com/MichaelDeBoey/gatsby-remark-embedder/blob/f2658708e6b2d292abef60d6b7fcc6be5965da18/src/Twitter.js#L4-L8